storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -2703,36 +2703,47 @@ static bool xtrabackup_read_info(char *filename) {
   FILE *fp;
   bool r = true;
   char mysql_server_version_str[30] = ""; /* 8.0.20.debug| 8.0.20 */
-  unsigned long mysql_server_version;
+  char xtrabackup_version_str[30] = "";   /* 8.0.20.debug| 8.0.20 */
+  unsigned long xb_server_version;
 
   fp = fopen(filename, "r");
   if (!fp) {
     xb::error() << "cannot open " << filename;
     return (false);
   }
-  /* skip uuid, name, tool_name, tool_command, tool_version, ibbackup_version */
-  for (int i = 0; i < 6; i++) {
+  /* skip uuid, name, tool_name, tool_command, tool_version */
+  for (int i = 0; i < 5; i++) {
     char c;
     do {
       c = fgetc(fp);
     } while (c != '\n');
   }
 
+  if (fscanf(fp, "ibbackup_version = %29s\n", xtrabackup_version_str) != 1) {
+    r = false;
+    goto end;
+  }
+  xb_backup_version =
+      xtrabackup::utils::get_version_number(xtrabackup_version_str);
+
+  DBUG_EXECUTE_IF("simulate_lower_version", xb_backup_version = 80030;);
+
   if (fscanf(fp, "server_version = %29s\n", mysql_server_version_str) != 1) {
     r = false;
     goto end;
   }
-  mysql_server_version =
+
+  xb_server_version =
       xtrabackup::utils::get_version_number(mysql_server_version_str);
 
-  ut_ad(mysql_server_version > 80000 && mysql_server_version < 90000);
-  if (mysql_server_version < 80019) {
+  ut_ad(xb_server_version > 80000 && xb_server_version < 90000);
```

Comment:
what happens with cfg_version if I take a backup from 8.0.22 server using PXB 8.0.30?

Reply:
nothing change in that part of code `cfg_version = IB_EXPORT_CFG_VERSION_V5;`

Reply:
I got confused with xb_server_version and xb_backup_version. Understand it now. Thanks.